js/tab.js

```diff
@@ -234,3 +237,7 @@ Tab.prototype = {
 
 };
 
+Tab.showTab = function(host, chan) {
+  var tab = document.querySelector('[data-host=' + host + '][data-channel=' + chan + ']');
```

Comment:
heh, I get a `SyntaxError: An invalid or illegal string was specified` on this line with `host` as `'irc.mozilla.org'` and `chan` as `'\n'`.  This is trying to select this element in the dom:

`<x-card style="background-color: rgb(26, 188, 156);" data-channel="\n" data-host="irc-mozilla-org" id="__1"></x-card>`

In the console, to select the correct card, I have to do: `document.querySelectorAll('[data-channel=\\\\n]')` haha 4 back slashes.